runtime/compiler/runtime/RuntimeAssumptions.hpp

```diff
@@ -180,6 +193,7 @@ class TR_PersistentClassInfo : public TR_Link0<TR_PersistentClassInfo>
    int32_t                             _nameLength;
    flags8_t                            _flags;
    flags8_t                            _shouldNotBeNewlyExtended; // one bit for each possible compilation thread
+   CCVResult                           _ccvResult;
```

Comment:
This is going to increase the size of each entry in the CHTable. We should encode it on 8-bits.
Also, maybe a add an inline comment for this field. CCV is not very descriptive.

Reply:
I _think_ because the enum only has 3 values, it does get allocated only 8 bits by the compiler; however, I can make it explicitly 8 bits.